trademapper/js/trademapper.mapper.js

```diff
@@ -116,9 +116,9 @@ define(["d3", "topojson", "worldmap", "disputedareas", "countrycentre"], functio
 	setupZoom: function() {
 		var moduleThis = this,
 		zoomed = function() {
-			var translate = d3.event.translate,
-				scale = d3.event.scale;
-			console.log("target: " + d3.target);
+      var translate = [d3.event.transform.x, d3.event.transform.y];
```

Comment:
Not sure why this looks unindented? Is that an issue with github, or (just checking) a tab/space mixup?

Reply:
Tab/space mixup, I'll fix.

Reply:
Fixed.